demos/tensorflow-image-tagging/src/demoapp/core.clj

```diff
@@ -0,0 +1,131 @@
+;;
+;;       Copyright (c) 2017 Two Sigma Investments, LP.
+;;       All Rights Reserved
+;;
+;;       THIS IS UNPUBLISHED PROPRIETARY SOURCE CODE OF
+;;       Two Sigma Investments, LP.
+;;
+;;       The copyright notice above does not evidence any
+;;       actual or intended publication of such source code.
+;;
+(ns demoapp.core
```

Comment:
Better namespace name please.

Reply:
Suggestions welcome. We left it as a renaming pass from you ;)

Reply:
I really don't mind the namespace now that it's in a better named directory.